<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FORM_ControlPort3</class>
 <widget class="QWidget" name="FORM_ControlPort3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*background-image: url(:/pic_de260/bg_01.png);*/
QLabel{
color: rgb(15, 175, 252);
font: 17pt &quot;WenQuanYi Zen Hei&quot;;
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>50</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>TakaoPGothic</family>
     <pointsize>30</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(252, 233, 79);
font: 30pt &quot;TakaoPGothic&quot;;</string>
   </property>
   <property name="text">
    <string>控制器I/O</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="portinfo">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>130</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 20pt &quot;WenQuanYi Zen Hei&quot;;</string>
   </property>
   <property name="text">
    <string>主控制器AI端口查询</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pageinfo">
   <property name="geometry">
    <rect>
     <x>1420</x>
     <y>130</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);
font: 20pt &quot;WenQuanYi Zen Hei&quot;;</string>
   </property>
   <property name="text">
    <string>第3/6页</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1290</x>
     <y>50</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/ico_260/915.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="text3_3">
   <property name="geometry">
    <rect>
     <x>814</x>
     <y>329</y>
     <width>225</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>215</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>AI26-驻车制动压力</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="value_4">
   <property name="geometry">
    <rect>
     <x>1046</x>
     <y>329</y>
     <width>120</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>35</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(115, 210, 22);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="text1">
   <property name="geometry">
    <rect>
     <x>814</x>
     <y>206</y>
     <width>225</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>215</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>AI1-液压油温</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="value_1">
   <property name="geometry">
    <rect>
     <x>1046</x>
     <y>206</y>
     <width>120</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>35</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(115, 210, 22);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="text2">
   <property name="geometry">
    <rect>
     <x>814</x>
     <y>247</y>
     <width>225</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>215</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>AI2-燃油油位</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="value_2">
   <property name="geometry">
    <rect>
     <x>1046</x>
     <y>247</y>
     <width>120</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>35</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(115, 210, 22);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="text3">
   <property name="geometry">
    <rect>
     <x>814</x>
     <y>288</y>
     <width>225</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>215</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>AI15-远光灯输入信号</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="value_3">
   <property name="geometry">
    <rect>
     <x>1046</x>
     <y>288</y>
     <width>120</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>35</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(115, 210, 22);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
